src/tools/schema2base.ts

```diff
@@ -29,18 +30,27 @@ export abstract class Schema2Base {
   }
 
   private async processFile(src: string) {
+    const paths = {
+      root: '../..',
+      map: {
+        'https://$build/': `../../lib/build/`,
```

Comment:
I like having the build mapping, but I think that it may be incorrect? I thought we discussed aliasing `$build` with `bazel-bin`?